matlab/test/arrow/type/tDate64Type.m

```diff
@@ -15,7 +15,9 @@
 % implied.  See the License for the specific language governing
 % permissions and limitations under the License.
 
-classdef tDate64Type < hFixedWidthType
+% Modified test class for arrow.type.Date64Type and arrow.date64
```

Comment:
This line can be removed, since there is already a comment at the top of the file describing the class.